README.md

```diff
@@ -38,8 +38,29 @@ The repo can be cloned using
 
 
 ## Configure static code analysis tools
-- Checksyle: used to enforce Java coding convenions and formatting specified in `/config/checkstyle/checkstyle.xml`.
+- git hook: used to enforce Java coding conventions and formatting specified in `/config/checkstyle/checkstyle.xml`. And linting for JS. 
 Configuration guide is available at https://github.com/DeloitteDigitalAT/core-banking/blob/master/config/checkstyle/README.md
 - Sonar
 - TBD
 
+### Install git pre-commit hook for checkstyle, ESlint and Prettier
```

Comment:
For checkstyle I have already created a separate file for instructions at https://github.com/DeloitteDigitalAT/core-banking/blob/master/config/checkstyle/README.md and only referencing it here, to avoid growing main README with 100s of lines about config and troubleshooting instructions. Could you please do the same for this section and also double-check what is in the already existing file since I do believe there is some overlapping?

Reply:
Yeah, I edited the linked readme as well.
But just added a small excerpt here for quick setup, cause I feel the basic setup at least should be in the main readme to get started with the codebase.
I can take it out as well if the readme is too long.

Reply:
OK, "small" is relative of course :), but having a small excerpt is fine, sure, as long as there is no contradictory information here and in the other checkstyle readme.

Reply:
It's fine, I removed it and made it a bit more clear. 😄